src/js/components/FormField/FormField.js

```diff
@@ -72,6 +72,7 @@ class FormFieldContent extends Component {
             update(name, event.target.checked);
             if (onChange) onChange(event);
           }}
+          aria-invalid={invalid || undefined}
```

Comment:
I don't think `aria-invalid` should accept `undefined`, from what I see on its docs it should be set to `false` instead. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute

Reply:
`undefined` means we will remove it from the DOM, which matches the initial state.